tests/unit-tests/api/wc-blocks/products-attributes-terms.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @package WooCommerce\Tests\API
+ */
+
+/**
+ * Product Controller "products attributes terms" REST API Test
+ *
+ * @since 3.6.0
+ */
+class WC_Tests_API_Products_Attributes_Terms_Controller extends WC_REST_Unit_Test_Case {
+
+	/**
+	 * Endpoints.
+	 *
+	 * @var string
+	 */
+	protected $endpoint = '/wc-blocks/v1';
+
+	/**
+	 * Setup test products data. Called before every test.
+	 *
+	 * @since 3.6.0
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->user = $this->factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+        );
+        
+        $this->editor = $this->factory->user->create(
```

Comment:
This endpoint allows users with `edit_posts` capability, so we could test with contributors.